src/langs/en.json

```
@@ -34,5 +34,17 @@
   "MAP_BUTTON_DISABLED_TEXT": "This item does not have a geolocation",
   "FROM_SHORTCUT_BUTTON_TEXT": "Back to {{name}}",
   "GUEST_LIMITATION_TEXT": "You are currently using Graasp with the guest account <1>{{name}}</1>. In order to use all features of Graasp, you have to log out and create a Graasp account.",
-  "GUEST_SIGN_OUT_BUTTON": "Log out and Create a Graasp account"
+  "GUEST_SIGN_OUT_BUTTON": "Log out and Create a Graasp account",
+  "ENROLL_SCREEN_TITLE": "Enroll to {{name}}",
+  "ENROLL_SCREEN_DESCRIPTION": "This resource allows access with your Graasp account. To access this course, simply click Enroll",
```

It is always a course? If not, maybe we could update it:

```suggestion
  "ENROLL_SCREEN_DESCRIPTION": "This resource allows access with your Graasp account. To access it, simply click Enroll",
```